constants/constants.go

```diff
@@ -39,6 +39,7 @@ var (
 	globalS3Endpoint               = getEnvVar("BACKUP_GLOBALS3ENDPOINT", "")
 	globalSecretAccessKey          = getEnvVar("BACKUP_GLOBALSECRETACCESSKEY", "")
 	globalStatsURL                 = getEnvVar("BACKUP_GLOBALSTATSURL", "")
+	metricBind                     = getEnvVar("BACKUP_METRICBIND", ":8080")
```

Comment:
`bind` and `address` or `addr` go together
```suggestion
	metricBindAddr                     = getEnvVar("BACKUP_METRIC_BINDADDRESS", ":8080")
```